opendebates/models.py

```diff
@@ -45,6 +45,9 @@ class SiteMode(CachingMixin, models.Model):
     announcement_link = models.URLField(null=True, blank=True)
     announcement_page_regex = models.CharField(max_length=255, null=True, blank=True)
 
+    banner_header_title = models.TextField(default=u'Welcome to the<br>Open Debate')
```

Comment:
This is probably fine given the temporary nature of these sites, but I did want to note that, if there are any existing sites out there you want to support (and upgrade), their `banner_header_title` and `banner_header_copy` will be lost and set to these defaults upon application of this migration. If that's an issue, a data migration could be added to set the title & copy for any existing SiteModes immediately after this migration is applied.